apps/server/src/infra/database/management/database-management.service.ts

```diff
@@ -26,7 +26,6 @@ export class DatabaseManagementService {
 		const collection = this.getDatabaseCollection(collectionName);
 		const { insertedCount } = await collection.insertMany(jsonDocuments as BaseEntity[], {
 			forceServerObjectId: true,
-			bypassDocumentValidation: true,
```

Comment:
Why do you remove this line? ( Please write it down for the others)

Reply:
There was a very short explanation in the PR description. I added a bit more context.